lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -16,6 +16,7 @@
     - retrieve secrets from HashiCorp's vault
   notes:
     - Due to a current limitation in the HVAC library there won't necessarily be an error if a bad endpoint is specified.
+    - As of Ansible 2.8, if you use KV v2 now you get only the last secret from a path.
```

Comment:
Is the 2.8 here still correct, or does this needs to be updated to 2.10 (what current `devel` will become)?

Reply:
Or do you plan to backport this to 2.8? In that case, it should be more precise; the next 2.8.x version will be 2.8.6, though it might already be too late for it; @abadger until when will backports for 2.8.6 be merged?

In case you plan to backport it: what about backwards compatibility? Does this break existing playbooks/roles which were using the old v2 approach from the samples below? If yes, this can't be backported, and you also need a [2.10 porting guide](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guides/porting_guide_2.10.rst) entry.

Reply:
Hi @felixfontein -  I'll have to let @beudbeud respond to the 2.8 comment feedback.  (I'm not sure if that comment is required)

I was planning on back porting to 2.8 & 2.9.

I've tested the patch against both v1 and v2 KV stores and it works.  My view is that it won't break backward compatibility (as far as I've tested).

Reply:
@Im0 For me is okay to backport in 2.8 and 2.9

Reply:
@beudbeud I think Felix was asking if this was correct: `As of Ansible 2.8, if you use KV v2 now you get only the last secret from a path.`